cookbooks/private-chef/libraries/opc_users.rb

```diff
@@ -46,6 +46,7 @@ def create_orgs_and_users(organizations, user_root, knife_opc_cmd, domainname)
           execute "create_org_#{orgname}" do
             command "#{knife_opc_cmd} org create #{orgname} #{orgname} -f #{user_root}/#{orgname}-validator.pem"
             action :run
+            returns [0, 100]
```

Comment:
Hey Mark - this is a curious change.   Still going to merge, but I'm reminding myself to ask you about it next week :)